### DISCUSSION TITLE
Daily 27/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#303] [Daily]
- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve probando como hacer lo de sellenium para testing y revisando documentación.
Hoy voy a hacer las tareas que tengo asignadas, hacer cambios en documentación y diagrama de clases, revisar la app por si tiene fallas y voy a intentar hacer el video que pidieron de accesibilidad, también de testing.

**Comment by marceloedelgado** [comment]
Despues de varios dias estudiando y tratando dsin exito mostrar los datos  reales de los usuarios finalmente lo logre en el dia de hoy sin errores reformar el codigo para las 2 Clases ProfesionalAdapter y CatalogoActivity necesarios para mostrar las tarjetas en el catalogo
Hoy continuaré con el diseño de las tarjetas en catalogo_activity.xml y tarjeta_profesional.xml

**Comment by MarielaGomez** [comment]
Ayer tuve clases de ingles donde rendí la ultima evidencia oral, también estuve poniéndome al día sobre testing y lo que agregó la profesora como tarea para agregar al proyecto

**Comment by BraianTroncoso** [comment]
Hoy seguí realizando las tareas, estuve también ayudando para que puedan resolver algunas. Seguí testeando la app para encontrar bugs y agregarlas como respectivas issues.
Se han acumulado varias por la falta de trabajo colaborativo de todos modos se hará hasta donde se llegue.
Asistí a Ingles y rendí el Oral.
